src/test/java/org/fundacionjala/pivotal/cucumber/steps/CommonSteps.java

```diff
@@ -0,0 +1,38 @@
+package org.fundacionjala.pivotal.cucumber.steps;
+
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.When;
+import org.fundacionjala.core.util.Environment;
+import org.fundacionjala.pivotal.pageobjects.dashboard.Dashboard;
+import org.fundacionjala.pivotal.pageobjects.dashboard.NewProjectForm;
+import org.fundacionjala.pivotal.pageobjects.login.SignInPage;
+
+/**
+ * Common steps.
+ */
+public class CommonSteps {
+    private Dashboard dashboard;
+    private NewProjectForm formPage;

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "formPage" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "formPage" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)